### PR TITLE
Adopt python-copier-template 1.3.0

### PR DESCRIPTION
**Pull request by evalott100**
@M-Chan 

Still to decide on the `dev` test runner format from https://github.com/DiamondLightSource/python-copier-template/issues/122

The python utility `rst2myst` came in handy for a couple of files here, I'd imagine if you had many rst docs it would make things much easier. Might be worth mentioning on copier template docs if there's a page on converting from old skeleton repos.

### CODE REVIEW

**Comment by evalott100** [comment]
Docs publishing working fine here: https://pandablocks.github.io/PandABlocks-ioc/98_merge/index.html

File: src/pandablocks_ioc/__init__.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
-from importlib.metadata import version
+from importlib.metadata import version  # noqa
 
 from .ioc import create_softioc
 
-__version__ = version("PandABlocks-ioc")
+__version__ = version("pandablocks-ioc")
```
Comment (coretl) [inline src/pandablocks_ioc/__init__.py:5]:
```suggestion
from ._version import __version__
```

File: src/pandablocks_ioc/__init__.py [hunk 2]
```diff
@@ -1,8 +1,8 @@
-from importlib.metadata import version
+from importlib.metadata import version  # noqa
```
Comment (coretl) [inline src/pandablocks_ioc/__init__.py:1]:
```suggestion
```

**Review by coretl — CHANGES_REQUESTED** [review]
Please can you put the Dockerfile from copier in here instead of the epics-containers one, then put the action back to the copier one too?

**Comment by evalott100** [comment]
Changes were made [here](https://github.com/evalott100/PandABlocks-ioc/commit/082eedd667c7112d50e59829fd8a81775ce22bef) to try to use the published container CI in a test environment.

We decided to abandon this and use the same container CI as the copier template.

**Review by coretl — APPROVED** [review]

**Pull request merged**